src/Prompt.php

```diff
@@ -73,6 +74,12 @@ abstract public function value(): mixed;
      */
     public function prompt(): mixed
     {
+        static::$interactive ??= stream_isatty(STDIN);
```

Comment:
This will break existing Laravel applications when testing console commands in a CI environment. They won't fall back to the Symfony implementation, which is required for the test assertions to work.

It can be solved with a small change in the framework, but that won't help if users have this version of prompts with a slightly older version of the framework.

The solution might be:
* Release this as a breaking change (v0.2.0) to prevent current Laravel installations from installing it.
* Update the minimum required version of Prompts in the framework and add code to set interactivity mode when required.

Alternatively, we could default this to `true` for the time being to maintain existing behaviour and then make the breaking change later.